CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## Supported Release 4.0.0
+### Summary
+This release sees the enablement of rubocop, also an update to the lib directory with rubocop fixes and several other changes and fixes. Also a bump to the Puppet version compatibility and several Puppet language updates.
+
+#### Added
+- Updated README.md with example how to install MySQL Community Server 5.6 on Centos 7.3
+- Enabled Rubocop and addition of Rubocop fixes for /lib directory.
+
+#### Removed
+- Dropped legacy tests for db.pp.
+
+#### Changed
+- Replaced validate function calls with datatypes in db.pp.
+- Bumped recommended puppet version to between 4.7.0 and 6.0.0.
+- (MODULES-4604) Conditionalize name validation in mysql_grant type.
```

Comment:
The comment style should be like this now 
 https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/CHANGELOG.md#fixed 
@eputnam 👍  for pointing it out to me.

Reply:
@tphoney Updated to include links